src/lib/loaders/loaders_without_authentication/tineye.ts

```diff
@@ -0,0 +1,35 @@
+import { ReadStream } from "fs"
+import tineye from "lib/apis/tineye"
+
+type SearchArtworkByImageLoaderOptions = {
+  image: ReadStream
+  filename: string
+  contentType: string
+}
+
+export default (_opts) => {
+  const searchArtworkByImageLoader = async ({
+    image,
+    filename,
+    contentType,
+  }: SearchArtworkByImageLoaderOptions) => {
+    const response = await tineye("/search", {
+      method: "POST",
+      formData: {
+        image: {
+          value: image,
+          options: {
+            filename,
+            contentType,
+          },
+        },
+      },
+    })
+
+    return response.body
+  }
+
+  return {
+    searchArtworkByImageLoader,
```

Comment:
**suggestion**: Should we call this a `tineyeSearchLoader` or something similar? TinEye itself doesn't have any awareness of artworks so I wonder if we should avoid the term `artwork` here.

We'll likely need to add that artwork awareness in Metaphysics by adding an `artwork` field in `ReverseImageSearchResult`, backed by a gravity API loader.

Reply:
> Should we call this a tineyeSearchLoader or something similar?

Renamed